### DISCUSSION TITLE
Controls range (slider) - How to show also the value?

### DISCUSSION THREAD

**Discussion by CezarCobuz** [storybookjs/react-native#334] [Q&A] [answered]
For a given story that has the arg type as range (see below snippet) 

```
const argTypes = {
  value: {
    range: true,
    min: 0,
    max: 1000,
    step: 1
  }
}
```

Inside controls I see the name(value) and a slider for it but it would be useful to see the value itself (for example 562)
![range controls](https://user-images.githubusercontent.com/36707637/156043234-ae7b6e47-3dea-4798-b612-53291bb58794.png)

Would it be ok if I create a PR with adding that option ('showValue: boolean') inside `addons/ondevice-controls/src/types/Number.tsx`?

**Comment by dannyhw** [comment, marked as answer]
@CezarCobuz that sounds like a great idea. There is no particular reason not to show the number that I know of.


> Would it be ok if I create a PR with adding that option ('showValue: boolean') inside addons/ondevice-controls/src/types/Number.tsx?

Yes that sounds great, really appreciate all contributions. I think it could also just always show the number maybe. Lets see how it looks and decide after.

**Reply by CezarCobuz** [reply]
For the given example "value" is the name of the argument (or prop for the component). It makes more sense to keep that together with the number here is an example with 2 ranges for a random component with start and end

![range controls 2](https://user-images.githubusercontent.com/36707637/156051883-d3b4aa8c-3343-4c0d-851a-4b86e29deb68.png)

I will start a PR in a bit, I was just going over the contribution guide as we speak
